rasa/cli/interactive.py

```diff
@@ -105,6 +105,9 @@ def _set_not_required_args(args: argparse.Namespace) -> None:
     args.fixed_model_name = None
     args.store_uncompressed = False
     args.dry_run = False
+    args.skip_validation = True
```

Comment:
Shouldn't this be `False` by default?

Reply:
I didn't want to change the default behaviour for `rasa interactive` to validate by default only `rasa train` since thats what the issue in the ticket was about.

Reply:
Oh I see, all clear now! Cheers.